src/lib/components/core/Navigation.svelte

```diff
@@ -76,7 +76,10 @@
     {#each options as { text, href, external }}
       <a
         {href}
-        on:click|preventDefault={() => handleSelect(href)}
+        onclick={(event) => {
+          event.preventDefault();
+          () => handleSelect(href);
```

Comment:
```suggestion
          handleSelect(href);
```

This change is needed for navigation to work

Reply:
Thank you, I have fixed it now.